### PR TITLE
Update README badges

### PR DESCRIPTION
**Pull request by 3flex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,10 @@
 [![Maven Central](https://img.shields.io/maven-central/v/io.gitlab.arturbosch.detekt/detekt-cli)](https://search.maven.org/artifact/io.gitlab.arturbosch.detekt/detekt-cli)
 [![gradle plugin](https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/io/gitlab/arturbosch/detekt/io.gitlab.arturbosch.detekt.gradle.plugin/maven-metadata.xml.svg?label=Gradle&style=flat-square)](https://plugins.gradle.org/plugin/io.gitlab.arturbosch.detekt)
 
-![Pre Merge Checks](https://github.com/detekt/detekt/workflows/Pre%20Merge%20Checks/badge.svg?event=push)
+![Pre Merge Checks](https://github.com/detekt/detekt/workflows/Pre%20Merge%20Checks/badge.svg?branch=main)
 [![codecov](https://codecov.io/gh/detekt/detekt/branch/main/graph/badge.svg)](https://codecov.io/gh/detekt/detekt)
-[![CodeFactor](https://www.codefactor.io/repository/github/detekt/detekt/badge)](https://www.codefactor.io/repository/github/detekt/detekt)
```
Comment (schalkms) [inline README.md:10]: Is there even a need to have CodeFactor on this repo anymore? Essentially, we are dog-fooding the referenced detekt version in codefactor.

Comment (3flex) [reply README.md:10]: I agree it should be removed. I'll wait for at least one other to chime in before merging though.

Comment (chao2zhang) [reply README.md:10]: [CodeFactor](https://www.codefactor.io/repository/github/detekt/detekt/issues) was configured to gh-pages and the issues reported are all CSS. If we cannot change the configuration (I don't seem to have access), I agree that we should remove it.

**Review by BraisGabin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (schalkms) [reply README.md:10]: The webhook is configured in this repo under `settings/hooks`.
Furthermore, there's a codeclimate webhook that we also don't use.

**Pull request merged**